databricks/koalas/frame.py

```diff
@@ -225,21 +225,21 @@
 
 >>> df // 10
            angles  degrees
-circle          0       36
-triangle        0       18
-rectangle       0       36
+circle        0.0     36.0
+triangle      0.0     18.0
+rectangle     0.0     36.0
 
 >>> df.floordiv(10)
            angles  degrees
-circle          0       36
-triangle        0       18
-rectangle       0       36
+circle        0.0     36.0
+triangle      0.0     18.0
+rectangle     0.0     36.0
 
->>> df.rfloordiv(10)
+>>> df.rfloordiv(10)  # doctest: +SKIP
```

Comment:
This test is moved to `tests/test_dataframe.py::DataFrameTest::test_rfloordiv` because this behaviour can be different depends on the version of pandas.

In pandas < 0.24.0
```python
In pandas < 0.24.0

```python
>>> df.rfloordiv(10)
             angles  degrees
circle          inf      0.0
triangle   3.000000      0.0
rectangle  2.000000      0.0
```